### PR TITLE
fix rollbar issue: data is not defined

### PR DESCRIPTION
**Pull request by namrataukirde**
### What does this PR do?
BUG:  data is not defined error on rollbar.

NOTE: This PR fixes the rollbar issue.  Rollbar is not catching actual errors as  As data variable is undefined. 

https://rollbar.com/crossroads_it/browse.goodcity/items/615/

Please Review.

Thanks.

### CODE REVIEW

File: app/services/logger.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ export default Ember.Service.extend({
   rollbar: Ember.inject.service(),
 
   raiseError(errorData, reason, currentUser) {
+    let data;
     this.set("rollbar.currentUser", currentUser);
     this.get("rollbar").error(this.getError(reason), (data = errorData));
```
Comment (steveyken) [inline app/services/logger.js]:
```suggestion
    this.get("rollbar").error(this.getError(reason), (errorData));
```

Comment (patrixr) [inline app/services/logger.js]:
```suggestion
    this.set("rollbar.currentUser", currentUser);
    this.get("rollbar").error(this.getError(reason), errorData);
```

Comment (abhinav098) [reply app/services/logger.js]: @patrixr you got late, hahah

Comment (patrixr) [reply app/services/logger.js]: 🤣

**Pull request merged**